codeHF/dpl-config_run3.json

```diff
@@ -113,8 +115,10 @@
     "hf-task-dplus": {
         "d_selectionFlagDPlus": "0"
     },
-     "hf-task-lc": {
-        "d_selectionFlagLc": "0"
+    "hf-task-lc": {
+        "cutEtaCandMax": "-1.",
+        "d_selectionFlagLc": "0",
+        "d_selectionFlagLcbar": "0"
```

Comment:
Configuration of `hf-task-lc-mc` is missing completely.

Reply:
Thanks @vkucera . I pushed it.